agent/taskresource/envFiles/envfile.go

```diff
@@ -452,3 +458,50 @@ func (envfile *EnvironmentFileResource) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+// ReadEnvVarsFromEnvFiles reads the environment files that have been downloaded
+// and puts them into a list of maps
+func (envfile *EnvironmentFileResource) ReadEnvVarsFromEnvfiles() ([]interface{}, error) {
```

Comment:
why make this returns a list of interface{} rather than a map[string]string?

Reply:
because the list preserves the order of the env files while map does not

Reply:
oh, what i meant was why this is a list of interface{} rather than a list of map[string]string

Reply:
go was giving me problems about it so i was under the impression that i couldn't return a list of map but i must've done something wrong 😂 i can change it to make it more clear